<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTPPathAddDlg</class>
 <widget class="QDialog" name="FTPPathAddDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>增加FTP路径</string>
  </property>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>241</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UsrName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>132</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>admin</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Passwd">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>188</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>123456</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="OKBtn">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>241</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>141</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FtpServerHost">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.1.4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>81</y>
     <width>98</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FTP服务器端口:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>31</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ftp服务器IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>191</y>
     <width>38</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FtpServerPort">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>76</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
